packages/jest-transform/src/__tests__/__snapshots__/ScriptTransformer.test.ts.snap

```diff
@@ -347,7 +480,1304 @@ exports[`ScriptTransformer passes expected transform options to getCacheKeyAsync
+          "moduleNameMapper": Array [],
+          "modulePathIgnorePatterns": Array [],
+          "modulePaths": Array [],
+          "prettierPath": "prettier",
+          "resetMocks": false,
+          "resetModules": false,
+          "resolver": undefined,
+          "restoreMocks": false,
+          "rootDir": "/",
+          "roots": Array [],
+          "runner": "jest-runner",
+          "runtime": "/test_module_loader_path",
+          "sandboxInjectedGlobals": Array [],
+          "setupFiles": Array [],
+          "setupFilesAfterEnv": Array [],
+          "skipFilter": false,
+          "skipNodeResolution": false,
+          "slowTestThreshold": 5,
+          "snapshotFormat": Object {},
+          "snapshotResolver": undefined,
+          "snapshotSerializers": Array [],
+          "testEnvironment": "node",
+          "testEnvironmentOptions": Object {},
+          "testLocationInResults": false,
+          "testMatch": Array [],
+          "testPathIgnorePatterns": Array [],
+          "testRegex": Array [
+            "\\.test\\.js$",
+          ],
+          "testRunner": "jest-circus/runner",
+          "transform": Array [
+            Array [
+              "\\.js$",
+              "test_preprocessor",
+              Object {},
+            ],
+          ],
+          "transformIgnorePatterns": Array [
+            "/node_modules/",
+          ],
+          "unmockedModulePathPatterns": undefined,
+          "watchPathIgnorePatterns": Array [],
+        },
+        "configString": "{"automock":false,"cache":true,"cacheDirectory":"/cache/","clearMocks":false,"coveragePathIgnorePatterns":[],"cwd":"/test_root_dir/","detectLeaks":false,"detectOpenHandles":false,"errorOnDeprecated":false,"extensionsToTreatAsEsm":[],"fakeTimers":{"enableGlobally":false},"forceCoverageMatch":[],"globals":{},"haste":{},"id":"test","injectGlobals":true,"moduleDirectories":[],"moduleFileExtensions":["js"],"moduleNameMapper":[],"modulePathIgnorePatterns":[],"modulePaths":[],"prettierPath":"prettier","resetMocks":false,"resetModules":false,"restoreMocks":false,"rootDir":"/","roots":[],"runner":"jest-runner","runtime":"/test_module_loader_path","sandboxInjectedGlobals":[],"setupFiles":[],"setupFilesAfterEnv":[],"skipFilter":false,"skipNodeResolution":false,"slowTestThreshold":5,"snapshotFormat":{},"snapshotSerializers":[],"testEnvironment":"node","testEnvironmentOptions":{},"testLocationInResults":false,"testMatch":[],"testPathIgnorePatterns":[],"testRegex":["\\\\.test\\\\.js$"],"testRunner":"jest-circus/runner","transform":[["\\\\.js$","test_preprocessor",{}]],"transformIgnorePatterns":["/node_modules/"],"watchPathIgnorePatterns":[]}",
+        "coverageProvider": "babel",
+        "instrument": false,
+        "supportsDynamicImport": false,
+        "supportsExportNamespaceFrom": false,
+        "supportsStaticESM": false,
+        "supportsTopLevelAwait": false,
+        "transformerConfig": Object {},
+      },
+    ],
+  ],
+  "results": Array [
```

Comment:
The snapshots were updated to have more calls of `getCacheKey` now since we take it account in the script cache now.

Reply:
So thats why there are more calls now

Reply:
The performance penalty of hashing the contents every time might be too much? Maybe we can gate this feature behind an option flag for the transformer? It would be really nice to get this to work for https://github.com/salazarm/jspatch